tools/installers/install_fairseq.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euo pipefail
+
+MAKE=make
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+torch_15_plus=$(python3 <<EOF
+from distutils.version import LooseVersion as V
+import torch
+
+if V(torch.__version__) >= V("1.5"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+python_36_plus=$(python3 <<EOF
+from distutils.version import LooseVersion as V
+import sys
+
+if V(sys.version) >= V("3.6"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+cuda_version=$(python3 <<EOF
+import torch
+if torch.cuda.is_available():
+    version=torch.version.cuda.split(".")
+    # 10.1.aa -> 101
+    print(version[0] + version[1])
+else:
+    print("")
+EOF
+)
+echo "cuda_version=${cuda_version}"
+
+if "${torch_15_plus}" && "${python_36_plus}"; then
+
+    rm -rf fairseq
+    git clone https://github.com/simpleoier/fairseq.git
```

Comment:
What is the difference between your fairseq and official?

Installing from branch is a bad idea because we can't reflect the official updating and really confusing. 
I can't accept this. Please consider another way, e.g. override

Reply:
I made a few modifications for finetuning only a few layers in Wav2Vec2 model. I'll try some other ways for that. Thanks for the suggestion.

Reply:
@kamo-naoyuki I changed it a little bit, to install fairseq from the official repo. Can you please check it?

Reply:
Thanks! This is good.

We can also install fairseq using our setup.py, but as @simpleoier did, installing from install_fairseq.sh is better for me because I think fairseq should be optional since it's a little bit large module.

Just in case, could you note the commit id of your fairseq in install_fairseq.sh? Perhaps, they would change the API.

Reply:
Sure. Just did it!